spec/models/miq_queue_spec.rb

```diff
@@ -1,4 +1,4 @@
-describe MiqQueue do
+RSpec.describe MiqQueue do
```

Comment:
Are we changing this across the board?  If so, I think we should do that in its own PR.

Reply:
It's always been optional in rspec 3.x. I've been updating these as I go for some time. Not sure why it would need its own PR.

Reply:
Typically we do style changes like this in one big PR updating everything (for consistency across the code base), that's why I was suggesting it.

Reply:
Ok, removed.